llvm/test/CodeGen/NVPTX/load-store-256-addressing-invariant.ll

```diff
@@ -0,0 +1,549 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_100 -mattr=+ptx88 | FileCheck %s -check-prefixes=PTX
+; RUN: %if ptxas %{ llc < %s -march=nvptx64 -mcpu=sm_100 -mattr=+ptx88 | %ptxas-verify -arch=sm_100 %}
```

Comment:
you must use `%if ptxas-12.9` here and in other tests that use ptx 8.8. Older ptxas versions will error out.